prediction/tensorflow/README.md

```diff
@@ -1,17 +1,67 @@
 # TensorFlow Estimator - Deploy model
 
 The purpose of this directory is to provide a sample for how you can deploy a
-TensorFlow trained model in AI Platform.
+TensorFlow trained model in AI Platform with GPU.
 
-*   Run the training example under /training/base/core/tensorflow using the
-    `aiplatform-submit-train-job.sh` or `local-train.sh` scripts.
-*   Run `aiplatform-deploy-model.sh`
+*   Run the training example under `/training/tensorflow/structured/base/scripts` using the
+    `train-cloud.sh` or `train-local.sh` scripts.
+*   Run `cloud-deploy-model-gpu.sh`
 
+## GPU support
 
-## Scripts:
+Now if you want to deploy a new model and use GPUs, now is as simple as 
+define the machine type and select which accelerator you want to use for
+your new model. 
+
+Upgrade to the latest version of Google Cloud SDK
+
+```
+gcloud components update
+```
+
+Define the machine-type which will be handling these requests. In this case we enabled a `n1-standard-4` which is a Standard machine type with 4 vCPUs and 15 GB of memory. The full list is available [here](https://cloud.google.com/compute/docs/machine-types). 
+
+After you upload to new gcloud SDK version you will see the --accelerator option available. 
+The type of the accelerator can only be one of the following: 
+
+```
+nvidia-tesla-k80
```

Comment:
Oh is there a docs link yet for this list?

Reply:
Not yet. Still in Alpha